Bottom: 93dc66a9364592da81d18f53aa39c3b705415730
Top:    3a7f70c8f9b38229a5c04063ad5ead2eaa940b93
Author: Ian Kent <raven@themaw.net>
Date:   2013-02-27 10:58:36 +0800

Refresh of autofs-5.0.7-add-symlink-pseudo-option.patch

---

diff --git a/man/auto.master.5.in b/man/auto.master.5.in
index 8007542..d0d582c 100644
--- a/man/auto.master.5.in
+++ b/man/auto.master.5.in
@@ -159,6 +159,13 @@ on individual map entries of both types. Bind mounting of NFS file
 systems can also be prevented for specific map entrys by adding the
 "port=" mount option to the entries.
 .TP
+.I "symlink"
+This option makes bind mounting use a symlink instead of an actual bind
+mount. It is an autofs specific option that is a pseudo mount option and
+so is given without a leading dash. It may be used with indirect map
+entries either in the master map (so it effects all map entries) or with
+individual map entries.
+.TP
 .I "\-r, \-\-random-multimount-selection"
 Enables the use of ramdom selection when choosing a host from a
 list of replicated servers. This option is applied to this mount
